tests/test_utils.py

```diff
@@ -194,3 +195,91 @@ def test_get_prefixes(self):
+                Record(prefix="new.b", prefix_synonyms=["b"], uri_prefix="https://example.org/b/"),
+                Record(prefix="new.c", prefix_synonyms=["c"], uri_prefix="https://example.org/c/"),
+            ]
+        )
+        msdf = MappingSetDataFrame(df=df, converter=converter)
+        msdf._standardize_df_references()
+        self.assertEqual(
+            ("new.a:1", "new.b:2", "new.c:3"),
+            tuple(df.iloc[0]),
+        )
+
+    def test_standardize_idempotent(self):
+        """Test standardizing leaves correct fields."""
+        metadata = {"license": "https://example.org/test-license"}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual({"license": "https://example.org/test-license"}, msdf.metadata)
+
+    def test_standardize_metadata_upgrade_multivalued_single(self):
+        """Test standardizing upgrades a string to a list."""
+        metadata = {"creator_id": "orcid:0000-0003-4423-4370"}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual({"creator_id": ["orcid:0000-0003-4423-4370"]}, msdf.metadata)
+
+    def test_standardize_metadata_upgrade_multivalued_multiple(self):
+        """Test standardizing upgrades a string to a list."""
+        metadata = {"creator_id": "orcid:0000-0003-4423-4370 | orcid:0000-0002-6601-2165"}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual(
+            {"creator_id": ["orcid:0000-0003-4423-4370", "orcid:0000-0002-6601-2165"]},
+            msdf.metadata,
+        )
+
+    def test_standardize_metadata_multivalued(self):
+        """Test standardizing upgrades a string to a list."""
+        metadata = {"creator_id": ["orcid:0000-0003-4423-4370", "orcid:0000-0002-6601-2165"]}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        msdf._standardize_metadata_references()
+        self.assertEqual(
+            {"creator_id": ["orcid:0000-0003-4423-4370", "orcid:0000-0002-6601-2165"]},
+            msdf.metadata,
+        )
+
+    def test_standardize_metadata_multivalued_type_error(self):
+        """Test raising on a  non-string, non-list object given to a multivalued slot."""
+        metadata = {"creator_id": object()}
+        msdf = MappingSetDataFrame(df=pd.DataFrame(), converter=Converter([]), metadata=metadata)
+        with self.assertRaises(TypeError):
+            msdf._standardize_metadata_references()
+
+    def test_standardize_error_non_list(self):
+        """Test that a type error is raised when metadata is presented that should not be a list."""
+        metadata = {"mapping_registry_id": ["https://example.org/r1", "https://example.org/r2"]}
```

Comment:
`mapping_registry_id` is not a mapping set element. Use `mapping_set_id` instead.